_data/willbarkoff/willbarkoff.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: William Barkoff
 institution: The Dalton School
-profile_pic: wbarkoff.png
+profile_pic: willbarkoff.jpg
 quote: "There's 10 types of ppl: ppl who get binary, ppl who don't, and ppl who didn't expect a ternary joke"
```

Comment:
I just noticed this! Could you please delete quotes and special character (`:`) from this line to maintain the same format?